### ISSUE TITLE
Unable to specify TLS ciphers for GrpcClientSettings

### ISSUE THREAD

**Issue by squadgazzz** [akka/akka-grpc#1431]
Akka gRPC 2.0.0.
I couldn't find a way to specify ciphers for GrpcClientSettings to create a gRPC client service with TLS support.

**Comment by raboof** [comment]
> to create a gRPC client service with TLS support

If you're OK with the JRE defaults, I think you shouldn't need any particular configuration to be able to perform HTTPS calls.

If you want more control, you can probably use `withSslContext` and perhaps `withTrustManager`. This could definitely use some documentation, though :)

**Comment by squadgazzz** [comment]
> > to create a gRPC client service with TLS support
> 
> If you're OK with the JRE defaults, I think you shouldn't need any particular configuration to be able to perform HTTPS calls.
> 
> If you want more control, you can probably use `withSslContext` and perhaps `withTrustManager`. This could definitely use some documentation, though :)

Yes, I do provide custom SslContext. But there's no way to provide custom ciphers. Client sends unsupported ciphers to my SSL provider. On the server-side, there's an option in ConnectionContext where I can specify any ciphers.

```
ConnectionContext.https(
  sslContext = mySslContext,
  enabledCipherSuites = myCiphers,
  enabledProtocols = myProtocols)
)
```

**Comment by raboof** [comment]
Ah, so those are in the SSLEngine, not the SSLContext - indeed it'd make sense to make that possible somehow.

For the akka-http backend, perhaps it'd make sense to allow bringing your own `HttpsConnectionContext` as well. For the Netty backend I'm not sure how this could be achieved.

**Comment by squadgazzz** [comment]
> Ah, so those are in the SSLEngine, not the SSLContext - indeed it'd make sense to make that possible somehow.
> 
> For the akka-http backend, perhaps it'd make sense to allow bringing your own `HttpsConnectionContext` as well. For the Netty backend I'm not sure how this could be achieved.

Netty provides this ability with their own `SslContextBuilder` for both client and server sides.

**Comment by BillyAutrey** [comment]
@raboof greetings!

Is there anything on the roadmap to provide a `HttpsConnectionContext` for an HTTP/2 client?  Seems like it's still missing.

**Comment by benthecarman** [comment]
Any updates on this? Have an application that can't be used without being able to set a `HttpsConnectionContext`
